### ISSUE TITLE
OptionalUnit - lone Unit statement

### ISSUE THREAD

**Issue by schalkms** [arturbosch/detekt#1047]
After the discussion in #1034 the `OptionalUnit` rule should flag the following case where a lone Unit statement is useless.
```Kotlin
fun f() {
  Unit // flag
  var one = 1
}
```
